operations/network/network_management_install_guide/aruba/management_interface.md

```diff
@@ -0,0 +1,61 @@
+# Management Interface 
+
+The management interface can be used to gain remote management access to the switch. The management interface is accessible using the “mgmt” VRF and is separate from the data plane interfaces, which are in the “default” VRF. 
+
+Alternatively, a loopback interface can be configured to be used as management interface.
+
+```
+Address Mode
+Admin State
+Mac Address
+IPv4 address/subnet-mask
+Default gateway IPv4
+IPv6 address/prefix
+IPv6 link local address/prefix: fe10::96f1:28ff:fe1d:a901/64
+Default gateway IPv6
+Primary Nameserver
+Secondary Nameserver
+:
+: 10.110.135.51
+: 10.110.135.52
+: dhcp
+: up
+: 94:f1:28:1d:a9:01
+: 10.93.61.227/21
+: 10.93.56.1
+```
+
+Relevant Configuration
+
+Enable/disable the management interface 
+
+```
+switch(config)# interface mgmt switch(config-if-mgmt)# no shutdown
```

Comment:
switch(config-if-mgmt)# no shutdown
should go on its own line

switch(config-if-mgmt)# shutdown
should go on its own line

Reply:
resolved.